test/fixtures/users.yml

```diff
@@ -8,7 +8,12 @@ client:
 staff:
   email: staff@staff.com
   password_digest: cc3a0280e4fc1415930899896574e118
-  role: staff
+  role: staff_member
+
+other_staff:
+  email: other_staff@other_staff.com
+  password_digest: cc3a0280e4fc1415930899896574e118
```

Comment:
Can we calculate this password digest on the fly with erb?

Maybe `BCrypt::Password.create("password")`